recipes/gtest/all/conanfile.py

```diff
@@ -79,15 +99,18 @@ def lazy_lt_semver(v1, v2):
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         extracted_dir = glob.glob("googletest-*/")[0]
-        os.rename(extracted_dir, self._source_subfolder)
+        shutil.copytree(extracted_dir, self.source_folder, dirs_exist_ok=True)
```

Comment:
```suggestion
        tools.rename(extracted_dir, self.source_folder, dirs_exist_ok=True)
```

Reply:
thanks fot that input I will change that.